bundles/org.eclipse.cdt.lsp/src/org/eclipse/cdt/lsp/editor/EditorMetadata.java

```diff
@@ -70,4 +85,16 @@ public interface EditorMetadata extends ConfigurationMetadata {
 			LspUiMessages.SaveActionsConfigurationPage_FormatEditedLines,
 			LspUiMessages.SaveActionsConfigurationPage_FormatEditedLines_description);
 
+	/**
+	 * Returns the default {@link List} of {@link PreferenceMetadata}
+	 *
+	 * @since 3.0
+	 */
+	List<PreferenceMetadata<?>> defaults = List.of(//
```

Comment:
@ruspl-afed to take advantage of #425 I added the defaults here too.

Reply:
Wrong link - should have been #431

Reply:
oops. I just merged #432

Reply:
sorry, I wasn't aware that you are planning change exactly in the same area @jonahgraham